kolibri/core/assets/src/views/k-textbox/index.vue

```diff
@@ -113,22 +105,37 @@
       },
     },
     methods: {
+      /**
+       * @private
+       */
       updateText() {
         // v-model is just a :value + @input
         /**
          * Emits input event with new value
          */
         this.$emit('input', this.currentText);
       },
+      /**
+       * @private
+       */
       reset() {
         this.$refs.textbox.reset();
       },
+      /**
+       * @private
+       */
       emitKeydown(e) {
         /**
-         * Emits keydown event with event
+         * Emits keydown event
          */
         this.$emit('keydown', e);
       },
+      /**
+       * Focuses on the textbox
+       */
+      focus() {
+        this.$refs.textbox.$el.querySelector('input').focus();
+      },
```

Comment:
it would be so nice if the default were `@private` and then only this method would be annotated with `@public`

I assume this is out of our control?

Reply:
Yes currently the vueodoc parser's default for props/methods/etc is public https://github.com/learningequality/vuedoc-parser/blob/master/lib/parser.js#L12

Not out of our control since we are using our fork. :wink:

Reply:
yeah working on a PR now :)

Reply:
okay, PR submitted: https://github.com/learningequality/vuedoc-parser/pull/1


* merge that PR
* `yarn`-update the dependency in the style_guide plugin
* add `methodsDefaultPrivate: true` to `options` in  vue-doc-loader.js

then you should be able to simply annotate `focus` with `@public` and remove all the `@private` annotations

Reply:
Pushed! :champagne: